src/test/java/com/ea/restaurant/service/impl/utils/fixtures/UserFixture.java

```diff
@@ -0,0 +1,35 @@
+package com.ea.restaurant.service.impl.utils.fixtures;
+
+import com.ea.restaurant.constants.Roles;
+import com.ea.restaurant.constants.Status;
+import com.ea.restaurant.constants.UserType;
+import com.ea.restaurant.entities.User;
+import java.time.Instant;
+import java.util.Optional;
+
+public final class UserFixture {
+
+  public static User buildUser(
+      Long id, String name, String lastName, String username, String password) {
+    var user = new User(id);
+    user.setName(name);
+    user.setLastName(lastName);
+    user.setUsername(username);
+    user.setPassword(password);
+    user.setRoles(Roles.ADMINISTRATOR);
+    user.setType(UserType.INTERNAL);
+    user.setEntityStatus(Status.ACTIVE);
+    user.setCreatedBy(1L);
+    user.setUpdatedBy(user.getCreatedBy());
+    user.setCreatedDate(Instant.now());
+    user.setUpdatedDate(user.getCreatedDate());
+
+    return user;
+  }
+
+  public static Optional<User> BuildOptionalUser(
```

Comment:
Use camel case to name this function

Reply:
Also you don't need this function at all

Reply:
I think you can delete it

Reply:
Done.